change: r42

golozara opened the pull request.

golozara committed:
Update renv.lock

golozara committed:
minor fixes

minor fixes: adding continuous age, excluding 1002, changing cvRepetitions

mdlavallee92 committed:
align pkg name with repo

msuchard commented:
The new `renv.lock` references packages that no longer exist (so dependencies can _not_ be loaded):

```
Warning: failed to find binary for 'ParallelLogger 3.1.0' in package repositories
Warning: failed to retrieve 'https://mran.microsoft.com/snapshot/2023-01-08/bin/macosx/big-sur-arm64/contrib/4.2/ParallelLogger_3.1.0.tgz' [error code 6]
Warning: failed to find source for 'ParallelLogger 3.1.0' in package repositories
Warning: failed to find binary for 'jsonlite 1.8.4' in package repositories
Warning: failed to retrieve 'https://mran.microsoft.com/snapshot/2023-01-08/bin/macosx/big-sur-arm64/contrib/4.2/jsonlite_1.8.4.tgz' [error code 6]
```

Also, there are WAY too many unnecessary packages in the `renv.lock` file.   `renv::status()` reports:

```
The following package(s) are no longer used in this project:
                                 _
  ps                               [1.7.3]
  rprojroot                        [2.0.3]
  desc                             [1.4.2]
  diffobj                          [0.3.5]
  later                            [1.3.0]
  processx                         [3.8.0]
  rematch2                         [2.1.2]
  brio                             [1.1.3]
  callr                            [3.7.3]
  pkgload                          [1.3.2]
  praise                           [1.0.0]
  promises                         [1.2.0.1]
  waldo                            [0.4.0]
  commonmark                       [1.9.0]
  curl                             [5.0.0]
  httpuv                           [1.6.9]
  lazyeval                         [0.2.2]
  mathjaxr                         [1.6-0]
  sourcetools                      [0.1.7-1]
  testthat                         [3.1.7]
  xtable                           [1.8-4]
  anytime                          [0.3.9]
  boot                             [1.3-28.1]
  coro                             [1.0.3]
  crosstalk                        [1.2.0]
  data.table                       [1.14.8]
  here                             [1.0.1]
  htmlwidgets                      [1.6.2]
  httr                             [1.4.5]
  metadat                          [1.2-0]
  minqa                            [1.2.5]
  nloptr                           [2.0.3]
  numDeriv                         [2016.8-1.1]
  pbapply                          [1.7-0]
  png                              [0.1-8]
  RcppTOML                         [0.2.2]
  reactR                           [0.4.4]
  timechange                       [0.2.0]
  CompQuadForm                     [1.4.3]
  lme4                             [1.1-32]
  memuse                           [4.2-3]
  metafor                          [4.0-0]
  plotly                           [4.10.1]
  polspline                        [1.1.22]
  pool                             [1.0.1]
  PRROC                            [1.3.1]
  reactable                        [0.4.4]
  reticulate                       [1.28]
  shinycssloaders                  [1.0.0]
  shinydashboard                   [0.7.2]
  shinyWidgets                     [0.7.6]
  tippy                            [0.1.0]
  torch                            [0.9.1]
  BeastJar                         [1.10.6]
  clock                            [0.6.1]
  coda                             [0.19-4]
  HDInterval                       [0.2.4]
  openxlsx                         [4.2.5.2]
  rateratio.test                   [1.1]
  torchopt                         [0.1.3]
  IterativeHardThresholding        [1.0.2]
  lightgbm                         [3.3.5]
  broom                            [1.0.4]
  MatrixModels                     [0.5-1]
  SparseM                          [1.81]
  abind                            [1.4-5]
  carData                          [3.0-5]
  nnet                             [7.3-18]
  pbkrtest                         [0.5.2]
  quantreg                         [5.94]
  car                              [3.1-2]
  corrplot                         [0.92]
  jpeg                             [0.1-10]
  markdown                         [1.5]
  cowplot                          [1.1.1]
  exactRankTests                   [0.8-35]
  ggrepel                          [0.9.3]
  ggsci                            [3.0.0]
  ggsignif                         [0.6.4]
  gridtext                         [0.1.5]
  km.ci                            [0.5-6]
  KMsurv                           [0.1-5]
  mvtnorm                          [1.1-3]
  polynom                          [1.4-1]
  rstatix                          [0.7.2]
  zoo                              [1.8-11]
  ggpubr                           [0.6.0]
  ggtext                           [0.1.2]
  maxstat                          [0.7-25]
  survMisc                         [0.5.6]
  survminer                        [0.4.9]
  CirceR                           [ohdsi/CirceR@v1.3.0]
  OhdsiShinyModules                [ohdsi/OhdsiShinyModules@v1.1.0]
  PatientLevelPrediction           [ohdsi/PatientLevelPrediction@v6.3.1]
  ResultModelManager               [ohdsi/ResultModelManager@v0.3.0]
  BigKnn                           [ohdsi/BigKnn@v1.0.2]
  Capr                             [ohdsi/Capr@v2.0.0]
  Characterization                 [ohdsi/Characterization@v0.1.1]
  CohortDiagnostics                [ohdsi/CohortDiagnostics@v3.1.2]
  CohortExplorer                   [ohdsi/CohortExplorer@v0.0.11]
  DataQualityDashboard             [ohdsi/DataQualityDashboard@v2.1.2]
  DeepPatientLevelPrediction       [ohdsi/DeepPatientLevelPrediction@v1.1.2]
  EnsemblePatientLevelPrediction   [ohdsi/EnsemblePatientLevelPrediction@v1.0.2]
  Eunomia                          [ohdsi/Eunomia@v1.0.2]
  Hydra                            [ohdsi/Hydra@v0.4.0]
  PhenotypeLibrary                 [ohdsi/PhenotypeLibrary@v3.12.0]
  PheValuator                      [ohdsi/PheValuator@v2.2.2]
  SelfControlledCaseSeries         [ohdsi/SelfControlledCaseSeries@v4.1.0]
  SelfControlledCohort             [ohdsi/SelfControlledCohort@v1.6.0]
  ShinyAppBuilder                  [ohdsi/ShinyAppBuilder@v1.1.1]
  Hades                            [ohdsi/Hades@v1.11.0]
```

and

```
The following package(s) are used in the project, but are not installed:

	devtools
	OhdsiRTools
	remotes
	styler
The following package(s) are used in the project, but are not installed:

	devtools
	OhdsiRTools
	remotes
	styler
```

mdlavallee92 committed:
rename files to repo ReproducibilitySodhi2023

mdlavallee92 committed:
update pkg name and file style via pkg maintenance

mdlavallee92 committed:
switch pkg Doc

mdlavallee92 committed:
add vignettes from CompEff Skeleton

mdlavallee92 committed:
add vignette docs from skel

mdlavallee92 committed:
comment out tasks done

mdlavallee92 committed:
update readme

mdlavallee92 committed:
fix lock file using Ohdsi Tools

mdlavallee92 committed:
fix desc to reflect study

mdlavallee92 committed:
more pkg name fixing

mdlavallee92 committed:
Merge pull request #2 from roux-ohdsi/r42

additional changes for r42

mdlavallee92 commented:
Hi @msuchard I attempted to fix some of the issues with what @golozara posted. Updated the renv.lock file using OhdsiRTools I think that is the fix you are referring to above. Also made sure the pkg name matched the repo

@golozara you need to add some details in the DESCRIPTION about the study as well as in the README. 

I am sure I missed something...please tag me to let me know

golozara commented:
> Hi @msuchard I attempted to fix some of the issues with what @golozara posted. Updated the renv.lock file using OhdsiRTools I think that is the fix you are referring to above. Also made sure the pkg name matched the repo
> 
> @golozara you need to add some details in the DESCRIPTION about the study as well as in the README.
> 
> I am sure I missed something...please tag me to let me know

fixed! Thanks
